libraries/chain/include/eosio/chain/backing_store/kv_context_rocksdb.hpp

```diff
@@ -14,21 +14,22 @@ namespace eosio { namespace chain {
 
    static account_name get_payer(const char* data) {
       account_name payer;
-      memcpy(&payer, data, kv_payer_size);
+      memcpy(&payer, data, kv_payer_size); // Before this method is called, data was checked to be at least kv_payer_size long
       return payer;
    }
 
    static const char* actual_value_start(const char* data) {
       return data + kv_payer_size;
    }
 
+   template <typename View, typename Iterator>
```

Comment:
If we decide to collapse `Iterator` into `View` in some way, this will also collapse

Reply:
I have collapsed them.